src/ForceTorqueMonitor.c

```diff
@@ -0,0 +1,164 @@
+// ForceTorqueMonitor.c
+
+// SPDX-FileCopyrightText: 2022-2023, Yaskawa America, Inc.
+// SPDX-FileCopyrightText: 2022-2023, Delft University of Technology
```

Comment:
I'm not really sure how the licensing stuff works. But I assume RobotCloud should be added here.